schema_forms/radiology.py

```diff
@@ -1,86 +1,21 @@
-
-
-def mammography(conn, cursor, file_number):
-    module_name = "mammography"
+def tomosynthesis(file_number):
```

Comment:
yes.. I had added the module to just copy paste the code into the form so that I can just re-use the question statements and copy the dict options for choices.